packages/cosmic-swingset/test/captp-fixture.js

```diff
@@ -10,29 +10,35 @@ const PORT = 7999;
 export { E };
 
 export async function makeFixture() {
+  const noisy = process.env.NOISY;
```

Comment:
https://github.com/Agoric/agoric-sdk/issues/2160#issuecomment-792348353

Reply:
https://github.com/Agoric/agoric-sdk/wiki/Code-Review-Checklist

> Object Capability Discipline - minimizing, if not eliminating, ambient authority (#2160)

Reply:
I moved this `process.env.NOISY` access up to `test-home.js`.